buildScript/dependencies.gradle

```diff
@@ -98,6 +98,9 @@ dependencies {
 
     // duckdb - SQL support for parquet, csv, and tsv files
     implementation 'org.duckdb:duckdb_jdbc:1.1.3'
+
+    // embedded redis server; version ~ 6.2
+    implementation 'com.github.codemonstur:embedded-redis:1.4.3'
```

Comment:
what is this for?

Reply:
This is a bundled Redis server that will automatically launch if no existing Redis server is found. By default, it is intended for standalone (single-instance) deployments.